kbase-extension/static/kbase/js/widgets/appWidgets2/input/dynamicDropdownInput.js

```diff
@@ -71,6 +71,10 @@ define([
             userId = runtime.userId(),
             eventListeners = [];
 
+        if (dd_options.query_on_empty_input === undefined) {
```

Comment:
This should be use a type test, like `typeof dd_options.query_on_empty_input === 'undefined'`.
I know, I've used a test against the global `undefined` value a million times, but I try not to now.

Reply:
Sure, I get it. It's safer. Changed.